pkg/scheduler/framework/plugins/default_registry.go

```diff
@@ -31,3 +34,45 @@ func NewDefaultRegistry() framework.Registry {
 		noop.Name: noop.New,
 	}
 }
+
+// FrameworkConfigProducerArgs contains arguments that are passed to the producer.
```

Comment:
It'd be good to comment that we may introduce more fields in the future. Right now, only `Weight` field.

Reply:
done